jib-cli/src/integration-test/java/com/google/cloud/tools/jib/cli/cli2/TestProject.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.cli2;
+
+import com.google.cloud.tools.jib.filesystem.DirectoryWalker;
+import com.google.common.io.Resources;
+import java.io.Closeable;
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.gradle.testkit.runner.BuildResult;
+import org.gradle.testkit.runner.GradleRunner;
+import org.gradle.util.GradleVersion;
+import org.junit.rules.TemporaryFolder;
+
+public class TestProject extends TemporaryFolder implements Closeable {
```

Comment:
I wish we had a way to share test utility classes across all projects. But let's not try to resolve this in this PR.

@loosebazooka